src/components/PeopleFilters.tsx

```diff
@@ -33,63 +73,41 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuriesArr.map(century => (
+              <SearchLink
+                data-cy="century"
+                key={century}
+                className={classNames('button mr-1', {
+                  'is-info': centuries.includes(century),
+                })}
+                params={toggleCenturies(century)}
+              >
+                {century}
+              </SearchLink>
+            ))}
```

Comment:
You are mapping over the centuriesArr array and rendering a SearchLink component for each item. This could potentially be a performance issue if the array is very large. Consider using a virtualized list to render only the items that are currently visible on the screen.